lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -47,11 +47,14 @@
 // styles together, so that card becomes more of
 // a styled container.
 .beta-card {
-  background-color: $white;
```

Comment:
It looks like we use this `.beta-card` [elsewhere](https://github.com/DoSomething/phoenix/blob/9205916675229463de9a2b8afca4dfbf4d7ff51d/lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js#L14). Should we add the `.-white` modifier there?